src/http/response.ts

```diff
@@ -44,22 +45,84 @@ export default class Response {
    */
   public status_code: number = Status.OK;
 
+  /**
+   * @description
+   *     A property to hold the path to the users views directory
+   *     from their project root
+   *
+   * @property string views_path
+   */
+  private views_path: string | undefined;
+
+  /**
+   * @description
+   *     The render method extracted from dejs
+   *
+   * @property any views_renderer
+   */
+  private template_engine: boolean | undefined = false;
+
   // FILE MARKER: CONSTRUCTOR //////////////////////////////////////////////////
 
   /**
    * @description
    *     Construct an object of this class.
    *
    * @param any request
+   *
+   * @param ResponseOptions options
+   *     See Drash.Interfaces.ResponseOptions
    */
-  constructor(request: any) {
+  constructor(request: any, options: ResponseOptions = {}) {
     this.request = request;
     this.headers = new Headers();
+    this.template_engine = options.template_engine;
+    this.views_path = options.views_path;
     this.headers.set("Content-Type", request.response_content_type);
   }
 
   // FILE MARKER: METHODS - PUBLIC /////////////////////////////////////////////
 
+  /**
+   * @description
+   *     Render html files. Can be used with Drash's template engine, basic HTML files
+   *     or HTML files with dynamic data, such as: `<p>{{ user.name }}</p>`.
+   *     This method will read a file based on the `views_path` and filename passed in
+   *
+   * @param any args
+   *
+   * @example
+   *     // if `views_path` is "/public/views",
+   *     // file to read is "/public/views/users/add.html"
+   *     const content = this.response.render('/users/add.html', { name: 'Drash' })
+   *     if (!content) throw new Error(...)
+   *     this.response.body = content
+   *
+   * @return string|boolean
+   *     The html content of the view, or false if the `views_path` is not set.
+   */
+  public render (...args: any): string|boolean {
+    if (!this.views_path) {
+      return false
+    }
+    const filename = this.views_path += args[0];
+    const data = args.length >= 2 ? args[1] : null
+    if (this.template_engine && data) {
+      const engine = new Drash.Compilers.TemplateEngine();
+      return engine.render(filename, data);
+    }
+    const fileContentsRaw = Deno.readFileSync(filename);
+    const decoder = new TextDecoder();
+    let decoded = decoder.decode(fileContentsRaw);
+    if (data) {
```

Comment:
does it change anything if Drash's template engine handles dynamic data?

Reply:
Addressed this in the below commit `Updated render method to not handle basic dynamic html`